server/src/authentication/auth.service.ts

```diff
@@ -15,8 +16,7 @@ export class AuthService {
         const user = await this.usersService.findByEmail(email);
 
         //TODO these passwords need to be hased in the database
```

Comment:
Can you just remove this to-do comment. That way we can search "TODO" later to see if we missed anything